kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -62,6 +62,7 @@ function _contentState(data) {
     available: data.available,
     files: data.files,
     progress,
+    breadcrumbs: [],
```

Comment:
Indeed, this was intentional - the breadcrumbs are no longer used in the content page (which was the only placed they were referenced from the content state), so I deleted them - apparently the breadcrumb component is not robust to the attribute being missing, however.